.styleci.yml

```diff
@@ -0,0 +1,8 @@
+enabled:
+
+disabled:
+  - align_double_arrow
```

Comment:
Symfony did it before but has changed it a year ago or so, to avoid those big diffs.

In the docs, we still use it as docs are about readability.